src/SWP/Bundle/ContentBundle/Rule/Applicator/ArticleRuleApplicator.php

```diff
@@ -77,6 +88,10 @@ public function apply(RuleInterface $rule, RuleSubjectInterface $subject)
 
         $subject->setTemplateName($configuration[$this->supportedKeys[1]]);
 
+        if ((bool) $configuration[$this->supportedKeys[2]]) {
```

Comment:
yeah, but when you add new element at start of array then it's hard to catch that bug.